chainer/training/updaters/__init__.py

```diff
@@ -0,0 +1,3 @@
+from chainer.training.updaters import multiprocess_parallel_updater  # NOQA
+
```

Comment:
I followed style of other `__init__.py`. They separate `import namespace` and `import classes`.